vault/activity_log_test.go

```diff
@@ -1326,129 +1328,55 @@ func setupActivityRecordsInStorage(t *testing.T, base time.Time, includeEntities
 		entityRecords = []*activity.EntityRecord{
 			{
 				EntityID:    "11111111-1111-1111-1111-111111111111",
-				NamespaceID: "root",
+				NamespaceID: namespace.RootNamespaceID,
 				Timestamp:   time.Now().Unix(),
 			},
 			{
 				EntityID:    "22222222-2222-2222-2222-222222222222",
-				NamespaceID: "root",
+				NamespaceID: namespace.RootNamespaceID,
 				Timestamp:   time.Now().Unix(),
 			},
 			{
 				EntityID:    "33333333-2222-2222-2222-222222222222",
-				NamespaceID: "root",
+				NamespaceID: namespace.RootNamespaceID,
 				Timestamp:   time.Now().Unix(),
 			},
 		}
+		if constants.IsEnterprise {
```

Comment:
it's probably better to put this in `vault/activity_log_ent_test.go`

Reply:
Why so?

Reply:
it's an enterprise specific test involving enterprise features (namespaces). that said, it does mean less code duplication by having this as an optional add-on, rather than two separate tests

what do we generally do in this situation?

Reply:
We went back and forth on some variations on this theme.  Typically we have two variations on the same test, one oss, one ent, or we have the test only in ent.  I think this approach is fine too, and has some advantages.  We should probably expand the wiki "Writing Tests: Models to Follow" to talk about this topic.